### PR TITLE
bump gl-native to 10.10.0-beta.1, common to 23.2.0-beta.1

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt [hunk 1]
```diff
@@ -594,4 +594,12 @@ internal class NativeMapImpl(val map: MapInterface) :
...
     map.unsubscribe(observer)
   }
+
+  override fun setRenderWorldCopies(renderWorldCopies: Boolean) {
+    map.renderWorldCopies = renderWorldCopies
+  }
+
+  override fun getRenderWorldCopies(): Boolean {
+    return map.renderWorldCopies
+  }
```
Comment (ank27) [inline sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt:604]: Can you add tests for these too.

**Comment by ank27** [comment]
Please add `public changelog` from gl-native release too.

File: sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt (hunk 1 above)
Comment (kiryldz) [reply sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt:604]: @alexander-kulikovskii if you leave them here - they're not exposed to the final user. Please reach out to gl-native folks to understand whether we want to expose them or not and should they be experimental or not

**Comment by alexander-kulikovskii** [comment]
> Please add `public changelog` from gl-native release too.

done

File: sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt (hunk 1 above)
Comment (alexander-kulikovskii) [reply sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt:604]: I'll expose it in separate PR

**Review by ank27 — APPROVED** [review]

**Review by yunikkk — APPROVED** [review]

**Review by kiryldz — APPROVED** [review]

**Pull request merged**